CMSgov/easi-app
# [NOREF] Update CEDAR Core system cache time from 5 minutes to 6 hours

# NOREF

Relates to CEDAR Jira ticket: https://jiraent.cms.gov/browse/CI-208

## Changes and Description

- Update CEDAR Core System Summary cache refresh interval from 5 minutes to be configurable in SSM parameters/environment variables.
- This is currently configured in each environment to be 6 hours at time of writing.

![image](https://user-images.githubusercontent.com/15203744/183704439-68931609-5c52-488f-a671-889f0ad1179d.png)

## How to test this change

- Cloning this locally and playing with the cache refresh time setting it to something like 5 seconds, or maybe 1 minute - just to see that it works as expected!
  - Do do this, add a line like this to your `.envrc.local` file!
```bash
export CEDAR_CACHE_INTERVAL="10s"
```

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
